src/components/data-table-v2/_data-table-v2-core.scss

```diff
@@ -170,3 +170,296 @@
+      padding: rem(10px) $spacing-md 0 $spacing-md;
+    }
+
+    //width for overflow menu columns
+    .#{$prefix}--table-column-menu {
+      width: rem(52px);
+      min-width: rem(52px);
+      padding-top: rem(8px);
+      padding-right: rem(8px);
+    }
+
+    .#{$prefix}--table-column-menu .#{$prefix}--overflow-menu[aria-expanded='false']:hover {
+      background: $ui-03;
+    }
+
+    //width for additional icon columns
+    .#{$prefix}--table-column-icon {
+      width: rem(32px);
+      min-width: rem(32px);
+      justify-content: center;
+      padding: 0;
+      padding-top: rem(8px);
+    }
+
+    .#{$prefix}--table-column-icon button {
+      width: rem(32px);
+      height: rem(32px);
+      vertical-align: middle;
+      background: transparent;
+      border: none;
+      outline: none;
+
+      &:focus:not([disabled]),
+      &:hover:not([disabled]) {
+        background: $ui-03;
+      }
+    }
+
+    .#{$prefix}--table-column-icon + .#{$prefix}--table-column-menu {
+      width: rem(40px);
+      min-width: rem(40px);
+      padding-left: 0;
+    }
+
+    // Overrides
+    .#{$prefix}--checkbox-label {
+      padding-left: rem(28px);
+    }
+
+    td .#{$prefix}--overflow-menu {
+      & svg {
+        opacity: 0;
+      }
+
+      &:hover svg,
+      &:focus svg {
+        opacity: 1;
+      }
+
+      &:focus {
+        outline: 0;
+        box-shadow: 0 2px 6px 0 rgba(0, 0, 0, 0.3);
+      }
+    }
+
+    tr:hover .#{$prefix}--overflow-menu svg {
+      opacity: 1;
+    }
+  }
+
+  //================================================
+  //NO SORT
+  //================================================
+  .#{$prefix}--data-table-v2--no-sort {
+    th:not(.#{$prefix}--table-column-checkbox):not(.#{$prefix}--table-column-menu):not(.#{$prefix}--table-column-icon):not(.#{$prefix}--table-expand-v2) {
+      padding-left: $spacing-sm;
+      padding-right: $spacing-sm;
+
+      &:first-of-type:not(.#{$prefix}--table-column-checkbox) {
+        padding-left: $spacing-lg;
+      }
+    }
+
+    td:not(.#{$prefix}--table-column-checkbox):not(.#{$prefix}--table-column-menu):not(.#{$prefix}--table-column-icon):not(.#{$prefix}--table-expand-v2) {
+      padding: rem(14px) $spacing-sm;
+
+      &:first-of-type:not(.#{$prefix}--table-column-checkbox) {
+        padding-left: $spacing-lg;
+      }
+    }
+
+    .#{$prefix}--table-sort-v2 {
+      padding: 0;
+    }
+  }
+
+  //================================================
+  //NUMERICAL COLUMN
+  //================================================
+  .#{$prefix}--data-table-v2--numerical-cell {
+    text-align: right;
+  }
+
+  //================================================
+  //DISABLED
+  //================================================
+  tbody tr[disabled] {
+    cursor: no-drop;
+
+    & td,
+    & td:hover {
+      color: $disabled;
+      background: $disabled-background-color;
+      cursor: no-drop;
+    }
+
+    & .#{$prefix}--table-expand-v2 button svg,
```

Comment:
Are there any class names that we could use instead of element references? (outside of svg) just to help protect it against markup changes